### DISCUSSION TITLE
The new Star feature

### DISCUSSION THREAD

**Discussion by cavo789** [community/community#8973] [Other Feature Feedback, Questions, & Ideas]
Thanks for allowing us to be able to sort our stars by assigning one ore more categories.

Is it possible to:

1. have the list of repositories not yet having a star category
2. when displaying the list of star repos to see the list of star categories
3. have a filter to be able to show only repos not yet in a category
4. have the option to sort the list of categories (by number of items and/or ascending)
5. to be able to add repos in a batch mode i.e. once filtered, a select all option then add to ... and specify a category

Thanks!

**Comment by brianlovin** [comment]
Love these suggestions, thanks @cavo789 — we have tracking issues for most of these, so I'll add your +1 where I can to make sure our team sees this. Overall, it seems like more control over search/sort/filter are popular requests!
